vmslave-config/ansible/roles/install-package-for-ci/tasks/main.yml

```diff
@@ -46,11 +46,17 @@
     executable: /usr/bin/expect
   when: ovftool_check.rc !=0
 
+- name: Check if apache2 is installed
```

Comment:
ok. again, will you plan to add some doc , so that people will be easier to understand those ?